config/base_config.go

```diff
@@ -19,6 +19,7 @@ package config
 
 import (
 	"bytes"
+	"github.com/apache/dubbo-go/common/yaml"
```

Comment:
hey, guy, move it to the 3rd import block.

Reply:
it's done